salt/orch/kubernetes.sls

```diff
@@ -127,6 +127,7 @@ reboot_setup:
     - require:
       - salt: kube_master_setup
 
+{% if salt['mine.get']('roles:kube-(master|minion)', 'caasp_fqdn', expr_form='grain_pcre').values()|length > 0 %}
```

Comment:
Wanted to play safe, but you are right, it wouldn't make sense as things would have failed far before we reach this point.